<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>552</width>
    <height>571</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Решение транспортной задачи</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0" colspan="2">
     <widget class="QTableWidget" name="tableWidget">
      <property name="tabletTracking">
       <bool>false</bool>
      </property>
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAsNeeded</enum>
      </property>
      <property name="dragEnabled">
       <bool>false</bool>
      </property>
      <property name="alternatingRowColors">
       <bool>false</bool>
      </property>
      <property name="showGrid">
       <bool>true</bool>
      </property>
      <property name="gridStyle">
       <enum>Qt::SolidLine</enum>
      </property>
      <property name="sortingEnabled">
       <bool>false</bool>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
      <property name="cornerButtonEnabled">
       <bool>true</bool>
      </property>
      <property name="rowCount">
       <number>4</number>
      </property>
      <property name="columnCount">
       <number>4</number>
      </property>
      <attribute name="horizontalHeaderVisible">
       <bool>true</bool>
      </attribute>
      <attribute name="horizontalHeaderCascadingSectionResizes">
       <bool>false</bool>
      </attribute>
      <attribute name="horizontalHeaderHighlightSections">
       <bool>true</bool>
      </attribute>
      <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
       <bool>false</bool>
      </attribute>
      <attribute name="verticalHeaderShowSortIndicator" stdset="0">
       <bool>false</bool>
      </attribute>
      <row>
       <property name="text">
        <string>A1</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>A2</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>A3</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>Потребность</string>
       </property>
      </row>
      <column>
       <property name="text">
        <string>B1</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>B2</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>B3</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Запас</string>
       </property>
      </column>
      <item row="0" column="0">
       <property name="text">
        <string>5</string>
       </property>
       <property name="foreground">
        <brush brushstyle="NoBrush">
         <color alpha="255">
          <red>0</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </brush>
       </property>
      </item>
      <item row="0" column="1">
       <property name="text">
        <string>3</string>
       </property>
      </item>
      <item row="0" column="2">
       <property name="text">
        <string>1</string>
       </property>
      </item>
      <item row="0" column="3">
       <property name="text">
        <string>10</string>
       </property>
      </item>
      <item row="1" column="0">
       <property name="text">
        <string>3</string>
       </property>
      </item>
      <item row="1" column="1">
       <property name="text">
        <string>2</string>
       </property>
      </item>
      <item row="1" column="2">
       <property name="text">
        <string>4</string>
       </property>
      </item>
      <item row="1" column="3">
       <property name="text">
        <string>20</string>
       </property>
      </item>
      <item row="2" column="0">
       <property name="text">
        <string>4</string>
       </property>
      </item>
      <item row="2" column="1">
       <property name="text">
        <string>1</string>
       </property>
      </item>
      <item row="2" column="2">
       <property name="text">
        <string>2</string>
       </property>
      </item>
      <item row="2" column="3">
       <property name="text">
        <string>30</string>
       </property>
      </item>
      <item row="3" column="0">
       <property name="text">
        <string>15</string>
       </property>
      </item>
      <item row="3" column="1">
       <property name="text">
        <string>20</string>
       </property>
      </item>
      <item row="3" column="2">
       <property name="text">
        <string>25</string>
       </property>
      </item>
      <item row="3" column="3">
       <property name="text">
        <string/>
       </property>
       <property name="background">
        <brush brushstyle="NoBrush">
         <color alpha="255">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </property>
       <property name="foreground">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>0</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </brush>
       </property>
       <property name="flags">
        <set>ItemIsSelectable|ItemIsEditable|ItemIsDragEnabled|ItemIsDropEnabled|ItemIsUserCheckable</set>
       </property>
      </item>
     </widget>
    </item>
    <item row="1" column="0">
     <layout class="QVBoxLayout" name="verticalLayout_5">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Выбор метода</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QRadioButton" name="radioButton_northWestCorner">
        <property name="text">
         <string>Метод северо-западного угла</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">buttonGroup</string>
        </attribute>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioButton_potetntialsMethod">
        <property name="text">
         <string>Метод потенциалов</string>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">buttonGroup</string>
        </attribute>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioButton_randomMethod">
        <property name="text">
         <string>Метод случайных чисел</string>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">buttonGroup</string>
        </attribute>
       </widget>
      </item>
      <item>
       <widget class="Line" name="line">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Интерфейс</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="checkBox_splitResultShow">
        <property name="text">
         <string>Раздельное отображение результата</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item row="1" column="1">
     <widget class="QPlainTextEdit" name="plainTextEdit_result">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="2" column="0" colspan="2">
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Количество поставщиков</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="spinBox_supply">
            <property name="minimum">
             <number>3</number>
            </property>
            <property name="maximum">
             <number>100</number>
            </property>
            <property name="value">
             <number>3</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Количество потребителей</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="spinBox_users">
            <property name="minimum">
             <number>3</number>
            </property>
            <property name="maximum">
             <number>100</number>
            </property>
            <property name="value">
             <number>3</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QPushButton" name="pushButton_create">
          <property name="text">
           <string>Создать таблицу</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_calculate">
          <property name="text">
           <string>Расчет</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>552</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Файл</string>
    </property>
    <addaction name="action"/>
    <addaction name="action_2"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>Программа</string>
    </property>
    <addaction name="action_3"/>
    <addaction name="action_4"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action">
   <property name="text">
    <string>Сохранить таблицу</string>
   </property>
  </action>
  <action name="action_2">
   <property name="text">
    <string>Загрузить таблицу</string>
   </property>
  </action>
  <action name="action_3">
   <property name="text">
    <string>Очистка</string>
   </property>
  </action>
  <action name="action_4">
   <property name="text">
    <string>Заполнить таблицу</string>
   </property>
  </action>
  <action name="action_5">
   <property name="text">
    <string>Сохранить расчеты</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="buttonGroup"/>
 </buttongroups>
</ui>
